README.md

```diff
@@ -1,147 +1,124 @@
 # KeyboardSupport
-[![CI Status](http://img.shields.io/travis/BottleRocketStudios/iOS-KeyboardSupport.svg?style=flat)](https://travis-ci.org/BottleRocketStudios/iOS-KeyboardSupport)
```

Comment:
We'll want to leave these parts of the readme intact.

Reply:
I copied over this by mistake. Going to put back in